<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Inventario de Edificios</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="QMdiArea" name="mdiArea">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="viewMode">
       <enum>QMdiArea::SubWindowView</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>&amp;Archivo</string>
    </property>
    <addaction name="actionNuevo"/>
    <addaction name="actionBuscar"/>
    <addaction name="separator"/>
    <addaction name="actionSalir"/>
   </widget>
   <widget class="QMenu" name="menuAyuda">
    <property name="title">
     <string>Ay&amp;uda</string>
    </property>
    <addaction name="actionAcerca_de"/>
   </widget>
   <widget class="QMenu" name="menuConfiguraci_n">
    <property name="title">
     <string>&amp;Configuración</string>
    </property>
    <addaction name="actionBarrios"/>
    <addaction name="actionEncuestadores"/>
   </widget>
   <addaction name="menuArchivo"/>
   <addaction name="menuConfiguraci_n"/>
   <addaction name="menuAyuda"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionNuevo">
   <property name="icon">
    <iconset resource="centrohistorico.qrc">
     <normaloff>:/icons/nuevo.png</normaloff>:/icons/nuevo.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Nuevo</string>
   </property>
   <property name="toolTip">
    <string>Nueva Edificación</string>
   </property>
   <property name="statusTip">
    <string>Agregar una nueva edificación</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionBuscar">
   <property name="icon">
    <iconset resource="centrohistorico.qrc">
     <normaloff>:/icons/buscar.png</normaloff>:/icons/buscar.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Buscar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionSalir">
   <property name="icon">
    <iconset resource="centrohistorico.qrc">
     <normaloff>:/icons/salir.png</normaloff>:/icons/salir.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Salir</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionAcerca_de">
   <property name="text">
    <string>Acerca &amp;de</string>
   </property>
  </action>
  <action name="actionBarrios">
   <property name="icon">
    <iconset resource="centrohistorico.qrc">
     <normaloff>:/icons/barrios.png</normaloff>:/icons/barrios.png</iconset>
   </property>
   <property name="text">
    <string>Barri&amp;os</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="actionEncuestadores">
   <property name="icon">
    <iconset resource="centrohistorico.qrc">
     <normaloff>:/icons/encuestadores.png</normaloff>:/icons/encuestadores.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Encuestadores</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="centrohistorico.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionSalir</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAcerca_de</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>showAbout()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionBuscar</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>showListado()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionBarrios</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>showBarrio()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionEncuestadores</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>showEncuesta()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionNuevo</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>showNuevo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>showAbout()</slot>
  <slot>showListado()</slot>
  <slot>showBarrio()</slot>
  <slot>showEncuesta()</slot>
  <slot>showNuevo()</slot>
 </slots>
</ui>
